src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -0,0 +1,285 @@
+(ns quo2.components.navigation.page-nav.view
+  (:require [quo2.components.avatars.group-avatar.view :as group-avatar]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.dropdowns.dropdown :as dropdown]
+            [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.navigation.page-nav.style :as style]
+            [quo2.theme :as theme]
```

Comment:
nit use `[quo2.theme :as quo.theme]` to avoid clashes with `theme` prop

Reply:
Totally agree, sorry, my IDE auto imports this one and I keep forgetting call it  quo.theme.